fix(converter): restrict ColourConverter to valid factory methods (#1516)
DisnakeDev/disnake: changelog/1488.bugfix.rst

> Fix ``ColourConverter`` accepting non-factory classmethods like ``holographic_style``, which returns a tuple instead of a single :class:`Colour`.

Reviewer: ```suggestion
Fix :class:`~disnake.ext.commands.ColourConverter` accepting ``classmethod`` names that do not return :class:`Colour`, for example ``holographic_style``.
```
